find_similar/tokenize.py

```diff
@@ -199,7 +200,7 @@ def tokenize(text: str, language: str, dictionary=None):
             word_normal_form = remove_part_speech(part_parse, dictionary=dictionary)
             if word_normal_form:
                 # remove stop words
-                if word_normal_form not in stop_words:
+                if remove_stopwords and word_normal_form not in stop_words or not remove_stopwords:
```

Comment:
Too complicated condition where `remove_stopwords` duplicated twice. I think two conditions will be better. `if remove_stopwords: ...`